python_apps/airtime_analyzer/setup.py

```diff
@@ -30,7 +30,7 @@
       scripts=['bin/airtime_analyzer'],
       install_requires=[
           'mutagen>=1.41.1', # got rid of specific version requirement 
-          'pika',
+          'pika~=1.1.0',
```

Comment:
What is the issue with higher pika versions?

Reply:
1.1.0 is the latest version which breaks the code as the API was changed:
**old:** `self._channel.basic_consume(self.msg_received_callback,queue=QUEUE, no_ack=False)`
**new:** `self._channel.basic_consume(QUEUE, self.msg_received_callback, auto_ack=False)`

It can just be `pika`, doing `~=1.1.0` just means we won't get unexpected package changes breaking things in the future. I've noticed a couple of those e.g. `mutagen` which was updated a few weeks ago. We actually installed LibreTime just before then and it worked fine, and I saw someone on discourse tried to install LibreTime just after mutagen dropped python2 support and they couldn't install. TBH I almost never to version numbers in my python requirements. Happy to take your lead here.